src/main/java/model/Car.java

```diff
@@ -0,0 +1,39 @@
+package model;
+
+public class Car implements Comparable<Car> {
+
+    private final Name name;
+    private final MovingCount movingCount;
+    private static final String DASH = "-";
+    private static final String STATUS_SEPARATOR = " : ";
+
+    public Car(Name name, MovingCount movingCount) {
+        this.name = name;
+        this.movingCount = movingCount;
+    }
+
+    public void move() {
+        this.movingCount.increase();
+    }
+
+    public boolean isSameMovingCount(Car otherCar) {
+        return this.movingCount.equals(otherCar.movingCount);
+    }
+
+    public String currentRacingStatus() {
+        String dash = DASH.repeat(movingCount.getMovingCount());
+        return this + STATUS_SEPARATOR + dash;
+    }
```

Comment:
Model 레이어에서 출력되는 형태를 결정하다보니 Model 이라는 레이어의 의미가 조금 흐릿해진 느낌이 듭니다!

Reply:
저도 @sami355-24님과 같은 의견입니다..!

Reply:
이 부분들도 명확하게 모델의 역할이 무엇인지 공부해봐야겠네요!! 감사합니다 ㅎㅎ

Reply:
이 부분은 getMovingCount만 넘겨준 다음에 String utils나 view쪽에서 처리해주면 더욱 좋을 것 같습니다!
```suggestion
    public String currentRacingStatus() {
        String dash = DASH.repeat(movingCount.getMovingCount());
        return this + STATUS_SEPARATOR + dash;
    }
```